glusterd2/commands/volumes/bricks-status.go

```diff
@@ -35,61 +50,18 @@ func checkBricksStatus(ctx transaction.TxnCtx) error {
 		ctx.Logger().WithError(err).Error("Failed to get volume information from store.")
 		return err
 	}
-
-	mtabEntries, err := getMounts()
+	brickStatuses, err := volume.CheckBricksStatus(vol)
 	if err != nil {
-		ctx.Logger().WithError(err).Error("Failed to read /etc/mtab file.")
+		ctx.Logger().WithError(err).Error("Failed to get brick status information from store.")
```

Comment:
The brick status information is not retrieved from store. You can remove the `from store` part.